src/main/resources/static/api.yaml

```diff
@@ -1335,6 +1335,54 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
 
+  /patients/profile/logintype:
```

Comment:
Why do we need a specific API? The only consumer of the info is a patient app. Why not just make it as part of patient profile API - /patients/me? If you think there is going to be more information in future for login - then we can add a root element "loginInfo" in the patient profile.

Reply:
App would decide which screen to show based on the login type (password screen or otp screen). If it is otp type of login, app needs to invoke the corresponding API to send otp to the user. Am not sure when would app invoke /patients/me. This API would be a pre-requisite for /sessions API.